src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -117,27 +114,25 @@ namespace winrt::TerminalApp::implementation
                                                      const bool initial,
                                                      const winrt::hstring cwd = L"");
 
-        // Normally, WindowName and WindowId would be
-        // WINRT_OBSERVABLE_PROPERTY's, but we want them to raise
-        // WindowNameForDisplay and WindowIdForDisplay instead
-        winrt::hstring WindowName() const noexcept;
-        winrt::fire_and_forget WindowName(const winrt::hstring& value);
-        uint64_t WindowId() const noexcept;
-        void WindowId(const uint64_t& value);
+        // For the sake of XAML binding:
```

Comment:
FYI: XAML can bind through `WindowProperties`

Reply:
Oh this comment might be dead - I tried doing it that way, but it _never_ updated when I raised PropertyChanged's on it. So I backed that out